src/dmd/libmscoff.d

```diff
@@ -312,13 +312,13 @@ final class LibMSCoff : Library
         om.base = cast(ubyte*)buf;
         om.length = cast(uint)buflen;
         om.offset = 0;
-        const(char)* n = global.params.preservePaths ? module_name : FileName.name(module_name); // remove path, but not extension
-        om.name = n.toDString();
+        const n = global.params.preservePaths ? module_name : FileName.name(module_name); // remove path, but not extension
```

Comment:
`n` is otherwise unused, just assign it directly to `om.name`

Reply:
Right